README.md

```diff
@@ -1,31 +1,62 @@
+Without going into a long history of interfaces to Web Services, this is not the first attempt to do so.  We can learn from CORBA, WSDL and WADL.  These specifications had good intentions but were limited by proprietary vendor-specific implementations, being bound to a specific programming language, and goals which were too open-ended.  In the end, they failed to gain traction.
+
+The OpenAPI Specification does not require you to rewrite your existing API.  It does not require binding any software to a service--the service being described may not even be yours.  It does, however, require the capabilities of the service be described in the structure of the OpenAPI Specification.  Not all services can be described by OpenAPI--this specification is not intended to cover every possible use-case of a REST API. The OpenAPI Specification does not define a specific development process such as design-first or code-first.  It does facilitate either technique by establishing clear interactions with a REST API.
+
+This GitHub project is the starting point for OpenAPI.
+Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like,
+and some general information regarding the project.
+
+## Current Version - 3.0
+
+The current version of the OpenAPI specification is 3.0 - and you can find it [here](versions/3.0.md).
+
+### [Previous Versions](versions/2.0.md)
+
+This repository also contains the OpenAPI Specification 2.0, which is identical to the Swagger 2.0 specification, 
+as well as the Swagger 1.2 and Swagger 2.0 specifications.
+
+Each folder in this repository, such as [examples](examples) and [schemas](schemas), should contain folders pertaining to the current and previous versions of the specification.
+
+## See it in Action
+
+If you just want to see it work, check out the [list of current examples](examples/v3.0).
+
+## Tools and Libraries
+
+Looking to see how you can create your own OpenAPI definition, present it or otherwise use it? Check out the growing
+[list of 3.0 Implementations](IMPLEMENTATIONS.md).
 
 ## Participation
 
-The OpenAPI Specification is a community driven, open project hosted by the Linux Foundation. The OAI encourages participation from individuals and companies alike. If you want to participate in the evolution of the OpenAPI Specification, consider taking the following actions:
+The current process for development of the OpenAPI Specification is described in 
+[Development Guidelines](DEVELOPMENT.md).
+Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the future specification, this branch will be merged to master.
```

Comment:
`[Technical Developer Community](https://www.openapis.org/participate/how-to-contribute/governance#TDC)`, not OAI Technical Contributors Board